deps/verifier/src/snp/mod.rs

```diff
@@ -307,6 +310,38 @@ fn get_common_name(cert: &x509::X509) -> Result<String> {
     Ok(e.data().as_utf8()?.to_string())
 }
 
+// Function to request vcek from KDS. Return vcek in der format.
+fn request_endorsement_key_kds(
+    att_report: AttestationReport,
+) -> Result<Vec<CertTableEntry>, anyhow::Error> {
```

Comment:
The function is specific to download the VCEK, so I think it could be rename to something more specific such as `fetch_vcek_from_kds()`

Reply:
Additionally, I think the function would be more self-contained if it returns `Result<Vec<u8>>`

Reply:
Changed. If appropriate, please mark this comment as Resolved.

Reply:
> Additionally, I think the function would be more self-contained if it returns `Result<Vec<u8>>`

Tried this and got a type mismatch error

Reply:
Btw I don't think you need `anyhow::Error` in in this return type.

Reply:
Fixed